create_marc_record.py

```diff
@@ -305,6 +308,146 @@ def add_discogs_data(base_record: Record, data: dict) -> Record:
+    else:
+        field_245 = Field(tag="245", indicators=["0", "0"], subfields=subfields_245)
+    base_record.add_ordered_field(field_245)
+
+    # 264 #1 $a [Place of publication not identified] : $b LABEL-INFO\LABEL\NAME, $c [DATE]
+    # For DATE - 1st 4 digits only
+    # If no label-info\label\name element, fill in $b with “[publisher not identified]”
+    # If there are multiple label-info\label\name elements, take only the first instance.
+    # If no date element, fill in $c with “[date of publication not identified]”
+    date_264 = data["full_json"]["date"][0:4]
+    if data["full_json"]["label-info-list"]:
+        label_info = data["full_json"]["label-info-list"][0]
+        if label_info["label"]["name"]:
+            publisher = label_info["label"]["name"]
+        else:
+            publisher = "[publisher not identified]"
+    else:
+        publisher = "[publisher not identified]"
+
+    # format: [Place of publication not identified] <space> <colon>
+    # $b  LABEL-INFO\LABEL\NAME <comma>
+    # $c <open square bracket>DATE<close square bracket>
+    subfields_264 = [
+        Subfield("a", "[Place of publication not identified] :"),
+        Subfield("b", publisher + ","),
+        Subfield("c", "[" + date_264 + "]"),
+    ]
+    field_264 = Field(tag="264", indicators=[" ", "1"], subfields=subfields_264)
+    base_record.add_ordered_field(field_264)
+
+    # 300 ## $a MEDIA\DISC-COUNT audio disc : $b digital ; $c 4 3/4 in.
+    # If no media\disc-count element, or if media\disc-count=0, fill in with “1.”
+    if data["full_json"]["medium-count"]:
+        qty = data["full_json"]["medium-count"]
+    # this qty is an int
+    if not qty or qty == 0:
+        qty = 1
+    if qty > 1:
+        qty_text = str(qty) + " audio discs :"
+    else:
+        qty_text = "1 audio disc :"
+
+    # format: $a MEDIA\DISC-COUNT audio disc <space> <colon>
+    # $b digital <space> <semicolon> $c 4 3/4 in.
+    subfields_300 = [
+        Subfield("a", qty_text),
+        Subfield("b", "digital ;"),
+        Subfield("c", "4 3/4 in."),
+    ]
+    field_300 = Field(tag="300", indicators=[" ", " "], subfields=subfields_300)
+    base_record.add_ordered_field(field_300)
+
+    # 500 ## $a Title from MusicBrainz database.
+    # same as 245 $a
+    subfields_500 = [Subfield("a", title_245)]
+    field_500 = Field(tag="500", indicators=[" ", " "], subfields=subfields_500)
+    base_record.add_ordered_field(field_500)
+
+    # 653 #6 $a TAGS\NAME
+    # include all tags in $a subfields
+    if data["full_json"]["tag-list"] != []:
+        subfields_653 = []
+        for tag in data["full_json"]["tag-list"]:
+            subfields_653.append(Subfield("a", tag["name"]))
+            field_653 = Field(tag="653", indicators=[" ", "6"], subfields=subfields_653)
+        base_record.add_ordered_field(field_653)
+
+    # 720 ## $a ARTIST\SORT-NAME.
+    artist_720 = data["full_json"]["artist-credit"][0]["artist"]["sort-name"]
```

Comment:
Does this MB element always exist?  The specs have a conditional where 720 is not created if no artist\sort-name.

Reply:
@ztucker4 confirmed the data always exists from MB, so this is fine.